.travis.yml

```diff
@@ -2,17 +2,34 @@ language: julia
 os:
   - osx
   - linux
+  - windows
+arch:
+  - x64
+  - x86
 julia:
-  - 1.0
-  - 1.1
+  - 1.0  # Latest LTS release
+  - 1    # Latest stable release
```

Comment:
`1` grabs the latest stable release?

Reply:
Yes it does. For our repos we kind of flip/flop between using this and explicitly stating 1.3 or 1.4. I think using 1 is preferable to repos that aren't updated frequently like this one.